data/101/762/977/101762977.geojson

```diff
@@ -98,7 +98,7 @@
         "Ammerbuch"
     ],
     "name:gsw_x_preferred":[
-        "\u00c3mmerbuch"
+        "Ammerbuch"
```

Comment:
As far as I can tell, this string should be "Ammerbuch", the existing name shouldn't be stored as an alt.